templates/partials/_publisher.html

```diff
@@ -0,0 +1,11 @@
+{% load static %}
+{% load books_extras %}
```

Comment:
I don't think we need to load static or books_extras for this page as they are not used?